services/QuillLMS/app/models/cron.rb

```diff
@@ -3,6 +3,8 @@
 class Cron
   # Configured in Heroku Scheduler to run every 10 minutes
   def self.interval_10_min
+    run_at_20_minute_mark if now.min == 20
+    run_at_50_minute_mark if now.min == 50
```

Comment:
I don't actually know how precise Heroku's "every ten minutes" is.  These might need to be ranges like `if now.min.between?(20, 29)` or something?

Reply:
That's a very astute point.  I do notice a lag sometimes with stuff kicking off in the logs.

Reply:
![Screen Shot 2022-02-01 at 11 54 25 AM](https://user-images.githubusercontent.com/2057805/152013775-d985464b-f4be-40cf-b256-372a58936475.png)

Your suspicions are confirmed

Reply:
Also in the heroku ui:
![Screen Shot 2022-02-01 at 11 56 01 AM](https://user-images.githubusercontent.com/2057805/152013972-1e601ab5-bc73-409b-a2f2-cdedad6de2df.png)

Reply:
Also noticed this warning too:
![Screen Shot 2022-02-01 at 12 20 20 PM](https://user-images.githubusercontent.com/2057805/152018204-a42e1015-b8bd-4090-a952-2073fa8b07d1.png)